contracts/infrastructure/dapp/ParaswapFilter.sol

```diff
@@ -153,9 +189,7 @@ contract ParaswapFilter is BaseFilter {
 
     function isValidMultiSwap(address _wallet, bytes calldata _data) internal view returns (bool) {
         (IParaswap.SellData memory sell) = abi.decode(_data[4:], (IParaswap.SellData));
-        return hasValidBeneficiary(_wallet, sell.beneficiary) &&
-            hasTradableToken(sell.path[sell.path.length - 1].to) && 
-            hasValidExchangeAdapters(sell.path);
+        return hasValidBeneficiary(_wallet, sell.beneficiary) && hasValidPath(sell.fromToken, sell.path);
     }
 
     function isValidSimpleSwap(address _wallet, address _augustus, bytes calldata _data) internal view returns (bool) {
```

Comment:
Is there any scenario that will use `simpleSwap` if the accepeted DEX are restricted to Uniswap (fork) and Paraswap Pools?

Reply:
Any single-path trade, e.g. a trade that uses 30% sushi, 30% uniV2, 40% paraswappool.

Reply:
I see, but then I'm not convinced by the implementation of `hasAuthorisedCallees`. In particular the use of `return authoriser.areAuthorised(_augustus, spenders, _callees, allData);`  since none of the DEXes has been added to the Registry.

Reply:
Yes but I thought we agreed that for our initial launch we would only support `swapOnUniswap[Fork]` and `multiSwap` and only add the callees necessary for `simpleSwap` later on. IMO `isValidSimpleSwap` can stay for now as it won't change, it will just return `false` until we add the filters for the callees.

Reply:
Ok. I thought we would remove `simpleSwap` until we really support it but it makes sense to keep the filter and just add the callees to the Registry.

As a side note, could we replace `_augustus` by `_wallet` in `return authoriser.areAuthorised(_augustus, spenders, _callees, allData);` to make the code a bit more readable ? I understand that Augustus is calling but in the end the call is being authorised for the wallet.

Reply:
I would find this a bit misleading as this variable is the `AugustusSwapper` contract, not the wallet address. Besides, we use both `_wallet` and `_augustus` in `isValidSimpleSwap` so what should we rename `_augustus` to in this function?

What about renaming it `_authorised` or just `_target`?